a3p-integration/proposals/z:acceptance/test-lib/sync-tools.test.js

```diff
@@ -0,0 +1,412 @@
+// @ts-check
+import test from 'ava';
+import '@endo/init/debug.js';
+import {
+  waitUntilAccountFunded,
+  waitUntilContractDeplyed,
+  waitUntilInvitationReceived,
+  waitUntilOfferResult,
+} from './sync-tools.js';
+
+const makeFakeFollow = () => {
+  let value = [[]];
+
+  const setValue = newValue => (value = newValue);
+  const follow = () => Promise.resolve(value);
+
+  return { setValue, follow };
+};
+
+const makeFakeBalanceQuery = () => {
+  let result = {
+    balances: [
+      {
+        denom: 'ubld',
+        amount: '364095061',
+      },
+      {
+        denom: 'uist',
+        amount: '2257215',
+      },
+    ],
+    pagination: {
+      next_key: null,
+      total: '0',
+    },
+  };
+
+  const setResult = newValue => (result = newValue);
+  const query = () => Promise.resolve(result);
+
+  return { setResult, query };
+};
+
+test.serial('wait until contract is deployed', async t => {
+  const { setValue, follow } = makeFakeFollow();
+  const waitP = waitUntilContractDeplyed(
+    'name',
+    {
+      follow,
+      setTimeout: globalThis.setTimeout,
+    },
+    {
+      maxRetries: 5,
+      retryIntervalMs: 1000,
+      log: t.log,
+      errorMessage: 'Contact not deplyed yet',
```

Comment:
typo
```suggestion
      errorMessage: 'Contract not deplyed yet',
```

Reply:
resolved in https://github.com/Agoric/agoric-sdk/pull/10171/commits/009d0c49dc2dbc457982e8effc048cbb1e496d36